enterprise/cmd/frontend/internal/registry/releases_db_test.go

```diff
@@ -22,7 +22,11 @@ func TestRegistryExtensionReleases(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	extensionID, err := (dbExtensions{}).Create(ctx, user.ID, 0, "x")
+	xExtensionID, err := (dbExtensions{}).Create(ctx, user.ID, 0, "x")
```

Comment:
```suggestion
	xExtensionID, err := dbExtensions{}.Create(ctx, user.ID, 0, "x")
```

The parentheses here should not be needed. (I think `gofmt -s` will fix those for you, though if we haven't run it before it might simplify other things too)